charts/octopus-deploy/values.yaml

```
@@ -104,6 +104,16 @@ octopus:
     annotations: {}
     labels: {}
 
+  proxy:
+    # the proxy to use for http requests
+    http:
+    # the proxy to use for https requests
+    https:
+    # the proxy to use for http and/or https requests, in case the above two values are not defined
+    all:
+    # a comma-separated list of hostnames that should be excluded from proxying
+    none:
```

It would be more "correct" if this was an array type, that then you can do a join in when using it. See https://github.com/OctopusDeploy/helm-charts/blob/main/charts/kubernetes-agent/templates/tentacle-deployment.yaml#L46-L47

An example 

Also can we rename this to be clearer?

```suggestion
    excludedHostnames: []
```